source/mir/las/sum.d

```diff
@@ -1988,6 +2053,28 @@ unittest
         assert(s3.array == c.array);
     }
 }
+unittest
+{
+    import core.simd;
+    import std.range: iota;
+    import std.array: array;
+
+    double[] ns = [9.0, 101.0];
+
+    foreach (n; ns)
+    {
+        double[] ar = iota(n).array;
+        double c = n * (n - 1) / 2; // gauss for n=100
+        double s0 = ar.sum!(Summation.pairwise);
+        double s1 = ar.sum!(Summation.kahan);
+        double s2 = ar.sum!(Summation.naive);
+        double s3 = ar.sum!(Summation.fast);
```

Comment:
> KBN and KB2?

sorry - somehow github displayed this at the wrong point. now it should be there ;-)